doc/cephfs/cephfs-mirroring.rst

```diff
@@ -439,7 +439,7 @@ This allows a user to add a non-existent directory for synchronization. The mirr
 will mark such a directory as failed and retry (less frequently). When the directory is
 created, the mirror daemon will clear the failed state upon successful synchronization.
 
-Adding a new snapshot or a new directory manually in the ``.snap`` directory of the
+Adding a new snapshot or a new directory in the ``.snap`` directory of the
 remote filesystem will result in failed status of the corresponding configured directory.
```

Comment:
Also mention that setting a snap-schedule on the remote flle system for directories that are being mirrored will cause the mirror daemon to report errors.

Reply:
@vshankar Updated